### PR TITLE
Added footer block and css

### PR DESCRIPTION
**Pull request by SachseInTheHub**

### CODE REVIEW

File: source/layouts/layout.erb [hunk 1]
```diff
@@ -13,19 +13,22 @@
     <%= javascript_include_tag  "all" %>
   </head>
   
-  <body class="<%= page_classes %>">
-    <header class="app-header">
-      
-    </header>  
-    
+  <body class="">
```
Comment (lukaszsagol) [inline source/layouts/layout.erb:16]: you don't need empty `class` here

File: source/layouts/layout.erb [hunk 2]
```diff
@@ -13,19 +13,22 @@
...
 		
     <footer class="site-footer">
+    <form class="sign-up__form" action="//madebymany.us3.list-manage.com/subscribe?u=bb6ef8c1b569c34510855551f&id=2c5505341e" method="post">
       <div class="grid site-width">
-        <div class="col-1-2">
-          An event by <a href="http://www.madebymany.com"><img src="images/mxm-logo.png" alt="Made by Many" /></a>
+        An event by <a href="http://www.madebymany.com"><img class="site-footer-logo" src="images/mxm-logo.png" alt="Made by Many" /></a>
+        </div>
+        <div class="sign-up__form__field">
+          <input placeholder="Join our mailing list" name="EMAIL" type="text" class="sign-up__form__email">
```
Comment (lukaszsagol) [inline source/layouts/layout.erb:25]: try using lowercase letters for `name`

Comment (SachseInTheHub) [inline source/layouts/layout.erb:25]: :+1:

File: source/layouts/layout.erb (hunk 1 above)
Comment (SachseInTheHub) [inline source/layouts/layout.erb:16]: :+1:

**Comment by lukaszsagol** [comment]
Merge merge merge :rocket:

**Comment by SachseInTheHub** [comment]
updated

**Pull request merged**